secrets/anki_kunny.age

```diff
@@ -0,0 +1,7 @@
+age-encryption.org/v1
+-> ssh-ed25519 zYX9Zw QVFeWK+PEfQCPOR93Q+ghMyKgyh0YHclEdSPFI6XKV4
+JOmJgbd3u7aX9ye/nD+Zm3g1+HGUomsayHLSf7/CtCg
+-> ssh-ed25519 NPk+5Q 7bp+jJhNXtB7wnXgsKvx+vqPcyXQqf+uZ6zDZ71EgTw
+CorOE35md0RKhAHSYfE+eqXLJ0vkAodSVuqBlLhy9do
+--- xSwqVpAgCaobeWC5VDKvvOrSNCbfT1R6SJwYWfaWGAU
+^N�*�����*5����J�G>S��gś��G�T�u����юI09��3S��.L>4�A�-6�U�k$�
```

Comment:
Sharing password signature (hash, whatever), is not the best practice
You essentially taking upon yourself to be the server's "registration" and "forgot password" buttons
I would like to ask you to think about scenarios where one of you would forgot/lost password
I would also like to urge you to search for nginx built in login mechanism, and if you can setup it as a proxy access manager to servers behind it

Reply:
Nginx does provide a basic auth indeed. And it's completely possible to do that(tho that still isn't fixing the main issue), the Anki sync server requires you to create a user like that, it surely can be done without it but it would mean making some sort of a webpage with ORM solution to verify logging in and restoring the password and that would involve forking the repo. I'm sorry, however it's the best solution for now that wouldn't require you to make a ton of additional work